app/schema.graphql

```diff
@@ -27,6 +40,16 @@ type DataQualityTimeSeries implements TimeSeries {
   data: [TimeSeriesDataPoint!]!
 }
 
+input DataSampler {
+  seed: Int! = 1234567890
+  nSamples: Int! = 1000
+}
```

Comment:
nit: the names of these make them feel like classes when they are just parameters. Let's discuss the naming.